packages/happner-cluster/test/_lib/hooks.js

```diff
@@ -141,12 +141,15 @@ async function stopClient(clients) {
 }
 
 async function startCluster(test, config) {
+  let firstFunction = config.functions[0];
```

Comment:
Can we come up with a more descriptive name instead of firstFunction?

Reply:
Better?

Reply:
packages/happner-cluster/test/_lib/hooks.js